### ISSUE TITLE
Need rules please!

### ISSUE THREAD

**Issue by baytech1** [RainLoop/rainloop-webmail#1448]
Would LOVE to see you add the ability to set rules to send new and existing emails into a folder

This is the one feature that MS has that we cannot live without.

Thank you for your consideration

Jim

**Comment by stenrose** [comment]
You already can, with sieve scripts (filters and vacation message). This requires your IMAP server to have sieve service enabled (if you're using Dovecot, see https://wiki2.dovecot.org/Pigeonhole/ManageSieve).

Sieve scripts is listed at https://www.rainloop.net/features/.

You can enable feature via Domains menu at Admin Panel. Select desired domain and click "Sieve configuration".

**Comment by M66B** [comment]
This should be implemented in an e-mail server, not in an e-mail client like Rainloop, because you want this to work too if the e-mail client is not running. My proposal is therefore to close this issue.

**Comment by stenrose** [comment]
@M66B, I agree with you. This should be implemented with the server not the client, and issue should therefore be closed. 

However, email client (web or desktop) may support managing server rules via ManageSieve protocol. And RainLoop does. You can enable filters feature via Domains menu at Admin Panel, if Sieve and ManageSieve is supported with your mail server. Select desired domain and click "Sieve configuration".

**Comment by phoenix1977** [comment]
The problem is many shared hosting provider doesn't support ManageSieve for Dovecot, i don't know why. But what's alternative?
